lesson45-props-state/homework/src/components/Blog/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import "./index.scss";
+
+const Blog = () => {
+  return (
+    <div className="blog">
+      <h2 className="blog__title title-h2">Blog</h2>
```

Comment:
create separate component that will be using different title and styles and be reusable